src/software/ai/hl/stp/play/stop_play_test.cpp

```diff
@@ -174,7 +177,7 @@ TEST_F(StopPlayTest, DISABLED_test_stop_play_ball_in_front_of_friendly_defense_a
     std::vector<ValidationFunction> non_terminating_validation_functions =
         stop_play_rules;
 
-    runTest(field, ball_state, friendly_robots, enemy_robots,
```

Comment:
This test (i.e. `test_stop_play_ball_in_front_of_friendly_defense_area`) is resulting in a SIGSEGV for me, though the comment regarding the bug also seems to be related to path planner. @jonathanlew maybe you can try seeing if this gets fixed in your PR aswell.